run-dask-process.sh

```diff
@@ -184,7 +179,7 @@ scheduler_pid=""
 worker_pid=""
 num_scheduler_tries=0
 
-function startScheduler {
+startScheduler () {
     mkdir -p $(dirname $SCHEDULER_FILE)
     echo "RUNNING: \"python -m distributed.cli.dask_scheduler $SCHEDULER_ARGS\"" > $SCHEDULER_LOG
     dask-scheduler $SCHEDULER_ARGS >> $SCHEDULER_LOG 2>&1 &
```

Comment:
```suggestion
    dask scheduler $SCHEDULER_ARGS >> $SCHEDULER_LOG 2>&1 &
```
From dask: dask-scheduler is deprecated and will be removed in a future release; use `dask scheduler` instead

Reply:
Thanks for catching that. @rlratzel you might also want to update this print
`echo "RUNNING: \"python -m distributed.cli.dask_scheduler $SCHEDULER_ARGS\"" > $SCHEDULER_LOG` since you are using `dask scheduler`. The same holds for starting the workers.

Reply:
Thanks, fixed in [de23228](https://github.com/rapidsai/multi-gpu-tools/pull/24/commits/de232283fa27014b41b2241f34b2575e5299204a).